RNAwesomeCardIO.podspec

```diff
@@ -0,0 +1,19 @@
+require 'json'
+package = JSON.parse(File.read('package.json'))
+
+Pod::Spec.new do |s|
+  s.name             = "RNAwesomeCardIO"
+  s.version          = package["version"]
+  s.summary          = package["description"]
+  s.requires_arc     = true
+  s.license          = { :type => package["license"] }
+  s.homepage         = package["homepage"]
+  s.authors          = { package["author"] => package["author"] }
```

Comment:
I'm afraid we had this and it's randomly generating
```
{ "First Name <email@email.tld>" => "First Name <email@email.tld>" }
```
and
```
{ "First Name" => "email@email.tld" }
```
So based on our testing, it should be:
```
{ package["author"]["name"] => package["author"]["email"] }
```